common/protob/messages-bitcoin.proto

```diff
@@ -414,6 +419,40 @@ message PrevOutput {
     optional uint32 decred_script_version = 3;      // only for Decred
 }
 
+/** Data type of a payment request for a set of outputs.
+ * @next TxRequest
+ */
+message TxAckPaymentRequest {
+    option (unstable) = true;
+
+    optional bytes nonce = 1;              // the nonce used in the signature computation
+    required string recipient_name = 2;    // merchant's name or coordinator's name in case of CoinJoin
+    repeated PaymentRequestMemo memos = 3; // any memos that were signed as part of the request
+    optional uint64 amount = 4;            // the sum of the external output amounts requested, required for non-CoinJoin
+    required bytes signature = 5;          // the trusted party's signature of the paymentRequestDigest
+
+    message PaymentRequestMemo {
+        optional TextMemo text_memo = 1;
+        optional RefundMemo refund_memo = 2;
+        optional CoinPurchaseMemo coin_purchase_memo = 3;
+    }
+
+    message TextMemo {
+        required string text = 1;      // plain-text note explaining the purpose of the payment request
+    }
+
+    message RefundMemo {
+        required string address = 1;   // the address where the payment should be refunded if necessary
```

Comment:
don't we also need the MAC (and slip44 maybe) here?

Reply:
https://github.com/trezor/trezor-firmware/pull/2069/commits/a1663be359d0e7b7a628f8653780d57f82f5ca37